nullaway/src/test/resources/com/uber/nullaway/testdata/NullAwayRxSupportPositiveCases.java

```diff
@@ -119,9 +119,9 @@ public Integer apply(NullableContainer<String> container) throws Exception {
   }
 
   private Observable<Integer> filterWithLambdaNullExpressionBody(Observable<String> observable) {
-      // BUG: Diagnostic contains: returning @Nullable expression from method with @NonNull return
-      // type
-      return observable.map(o -> perhaps() ? o : null).map(o -> o.length());
+    // BUG: Diagnostic contains: returning @Nullable expression from method with @NonNull return
+    // type
```

Comment:
actually we need to be careful here.  not sure if the EP test harness works correctly if the `// BUG` comment isn't exactly the line before the bad code.  Is this still giving the expected warning?  We should probably turn off GJF for `test/resources` test inputs.

Reply:
This works the same whether the comment is a single line or many. It does break if we add another line or another (non-blank) comment below. I assume EP is already set up to play nice with GJF, since it is a Google tool :)